src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -5,6 +5,7 @@ Rultor-DynamoKey: ${dynamo.key}
 Rultor-DynamoSecret: ${dynamo.secret}
 Rultor-GithubId: ${github.id}
 Rultor-GithubSecret: ${github.secret}
+Rultor-GithubThreshold: ${github.threshold}
```

Comment:
@cvrebert in this case, why do we want to have it configurable? let's just hardcode `Tv.HUNDRED` in Java and that's it. what do you think?